run/idp-sql/views/index.html

```diff
@@ -15,6 +15,7 @@
   <!-- [END cloudrun_end_user_firebase_scripts]-->
   <script src="/config.js"></script>
   <script src="/firebase.js"></script>
+  <link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>🐱</text></svg>">
```

Comment:
Wait a second, is this website operated by cats?

Reply:
😸🐾